flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/util/ObjectContainer.java

```diff
@@ -31,21 +31,25 @@
 @Internal
 public class ObjectContainer {
 
-    private final Object o;
+    private final Object object;
```

Comment:
I guess we do not need it if we use just ArrayList as mentioned above

Or did I miss anything?